mix.exs

```diff
@@ -36,6 +36,8 @@ defmodule Etl.MixProject do
       {:gen_stage, "~> 1.0"},
       {:brex_result, "~> 0.4.0"},
       {:timex, "~> 3.6"},
+      {:libgraph, "~> 0.13.3"},
+      {:uuid, "~> 1.1"},
```

Comment:
just a thought since this came up in Stargate: do we actually need the guarantees of UUID or would PUID be sufficient?

Reply:
not even sure I need UUIDs at all yet, I added it for 1 purpose, which I refactored to not need it.  I think I will need them for scatter/gather.  We can look a the requirements then.